arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1135,24 +1155,24 @@ Result RocksDBCollection::remove(
   // Check old revision:
   if (!options.ignoreRevs && slice.isObject()) {
     TRI_voc_rid_t expectedRevisionId = TRI_ExtractRevisionId(slice);
-    int res = checkRevision(trx, expectedRevisionId, oldRevisionId);
+    auto res = checkRevision(&trx, expectedRevisionId, oldRevisionId);
```

Comment:
why ?!

Reply:
int -> auto just in case the return changes later to reduce the number of changed lines
trx -> &trx because the function requires a transaction pointer